### PR TITLE
ceilometer-polling role was renamed (back) to ceilometer-central

### PR DESCRIPTION
**Pull request by jsuchome**
For both Cloud6 and Cloud7

See https://github.com/crowbar/crowbar-openstack/pull/373 and https://github.com/crowbar/crowbar-openstack/pull/376

### CODE REVIEW

**Comment by aplanas** [comment]
+1 Thanks!!!

**Comment by nicolasbock** [comment]
+1

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -2614,7 +2614,7 @@ function custom_configuration()
         ceilometer)
             local ceilometerservice="ceilometer-cagent"
             if iscloudver 6plus ; then
-                ceilometerservice="ceilometer-polling"
+                ceilometerservice="ceilometer-central"
```
Comment (vuntz) [inline scripts/qa_crowbarsetup.sh]: So technically, this should not be done for cloudsource=GM6. This might matter for maintenance QA.

**Comment by kbaikov** [comment]
+1

**Comment by aplanas** [comment]
Note: this is needed to make Cloud 6 gating green

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (jsuchome) [inline scripts/qa_crowbarsetup.sh]: @vuntz is it correct now? See 2nd commit

**Comment by aplanas** [comment]
Still +1

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -2614,7 +2614,11 @@ function custom_configuration()
...
+                if iscloudver 6 ; then
```
Comment (vuntz) [inline scripts/qa_crowbarsetup.sh]: I think you really need to look at the `$cloudsource` value; this check will work for develcloud6, which we don't want.

**Comment by vuntz** [comment]
+1

**Pull request merged**